common/chunkStatusLogger.go

```diff
@@ -257,6 +258,7 @@ type chunkStatusLogger struct {
 	unsavedEntries                  chan *chunkWaitState
 	flushDone                       chan struct{}
 	cpuMonitor                      CPUMonitor
+	done                            chan bool
```

Comment:
nit, let's make this a channel of empty struct{} as that seems to be a convention for signalling channels.